hp-epigraphs.tex

```diff
@@ -1,212 +1,219 @@
-% FIXME: Add an option to the main build to put these in their original positions
-\chapter{Disclaimers and Epigraphs}
+% \chapter{Disclaimer}
+\newpage
+\vspace*{4cm}
+\begin{center}
+Disclaimer:\\J. K. Rowling owns Harry Potter,\\and no-one owns the methods of rationality.
+\end{center}
+    
+% % FIXME: Add an option to the main build to put these in their original positions
+% \chapter{Disclaimers and Epigraphs}
 
-Included here are the disclaimers and epigraphs from the first thirty or so chapters. N.B. Chapter 11 in the original (Omake Files I–III) is Appendix B in this PDF.
+% Included here are the disclaimers and epigraphs from the first thirty or so chapters. N.B. Chapter 11 in the original (Omake Files I–III) is Appendix B in this PDF.
 
-{% start of spacing changes
-\setlength{\parindent}{0pt}
-\setlength{\parskip}{1em}
+% {% start of spacing changes
+% \setlength{\parindent}{0pt}
+% \setlength{\parskip}{1em}
 
-\filbreak
---- 1 ---
+% \filbreak
+% --- 1 ---
 
-Disclaimer: J. K. Rowling owns Harry Potter, and no-one owns the methods of rationality.
+% Disclaimer: J. K. Rowling owns Harry Potter, and no-one owns the methods of rationality.
 
-\begin{em}
-Beneath the moonlight glints a tiny fragment of silver, a fraction of a line{\ldots}
+% \begin{em}
+% Beneath the moonlight glints a tiny fragment of silver, a fraction of a line{\ldots}
 
-(black robes, falling)
+% (black robes, falling)
 
-{\ldots}blood spills out in litres, and someone screams a word.
-\end{em}
+% {\ldots}blood spills out in litres, and someone screams a word.
+% \end{em}
 
-\filbreak
---- 2 ---
+% \filbreak
+% --- 2 ---
 
-\texttt{\#include "stddisclaimer.h"}
+% \texttt{\#include "stddisclaimer.h"}
 
-"\emph{Of course it was my fault. There’s no-one else here who could be responsible for anything.}"
+% "\emph{Of course it was my fault. There’s no-one else here who could be responsible for anything.}"
```

Comment:
Curl the quotes even thought they're commented, so it will give the correct result if uncommented.